lab-matt/route/model-route.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
```

Comment:
These look great, but please remove the lab-08 code, create a second branch and add it in there.  The assignment specifies that your lab-08 submission must be on a second branch.